readme.rst

```diff
@@ -169,7 +146,7 @@ If the link is in-line, move the definition to a line of its own (e.g. among
 Exempt an entire file
 ^^^^^^^^^^^^^^^^^^^^^
 
-If it's necessary *and safe*, you can exempt a whole file from
+If it's necessary*and safe*, you can exempt a whole file from
```

Comment:
Nope.
(Where do those changes come from?)

Reply:
They came from me fixing the bullet points in a lazy way by replacing " *" with "*". Should all be fixed now.